HW2/P3/mandelbrot.pyx

```diff
@@ -17,6 +24,8 @@ cpdef mandelbrot(np.complex64_t [:, :] in_coords,
     cdef:
        int i, j, iter
        np.complex64_t c, z
+       np.float32_t [:,:] in_coords_r, in_coords_i
+       AVX.float8 c_r, c_i, z_r, z_i, counts, mask, z_temp_r, z_temp_i
```

Comment:
Great! That would be a little bit more clear if you could give variables either a short explanation/comments.